tests/test_views.py

```diff
@@ -524,6 +524,93 @@ def test_get_course_specific_consent(
     @mock.patch('enterprise.views.configuration_helpers')
     @mock.patch('enterprise.views.render', side_effect=fake_render)
     @mock.patch('enterprise.views.CourseApiClient')
+    @ddt.data(
+        (False, False),
+        (True, True),
+    )
+    @ddt.unpack
+    def test_get_course_specific_consent_ec_requires_account_level(
+            self,
+            enrollment_deferred,
+            supply_customer_uuid,
+            course_api_client_mock,
+            render_mock,  # pylint: disable=unused-argument
```

Comment:
@haikuginger Nit: [`render_mock_unused`](https://github.com/edx/edx-enterprise/blob/master/pylintrc#L155)? Then we can remove `# pylint: disable=unused-argument`.